website/pages/api-docs/secret/consul/index.mdx

```diff
@@ -149,12 +149,12 @@ $ curl \
 
 ### Sample request
 
-```sh
-curl \
-→     --request POST \
-→     --header "X-Vault-Token: ..."\
-→     --data @payload.json \
-→     http://127.0.0.1:8200/v1/consul/roles/example-role
```

Comment:
This is the only example where a character was removed (the `→` arrow). No other examples of `curl` included these arrows. I hope this bug fix can come along for the ride.